polybar-scripts/network-ipfinder/ipfinder.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
Use `#!/bin/sh` or describe why you need a bash.

Reply:
I've removed the bash builtin `declare` command.